src/Repository/EventRepository.php

```diff
@@ -208,6 +209,25 @@ public function findManagedBy(Adherent $referent): array
         return $qb->getQuery()->getResult();
     }
 
+    /**
+     * @return BaseEvent[]
+     */
+    public function findAllInDistrict(District $district): array
+    {
+        return $this->_em->getRepository(BaseEvent::class)->createQueryBuilder('e')
+            ->select('e', 'o')
+            ->leftJoin('e.organizer', 'o')
```

Comment:
is this leftJoin needed ?

Reply:
Yes, the organizer data is used in the events exporter.

Reply:
:zipper_mouth_face: :100: